src/modules/shared/components/MediaCardList/MediaCardList.tsx

```diff
@@ -125,6 +128,26 @@ const MediaCardList: FC<MediaCardListProps> = ({
 		];
 	};
 
+	const insertManyResultsTile = (tiles: ReactNode[]): ReactNode[] => {
+		if (showManyResultsTile) {
+			return [
+				...tiles.slice(0, MANY_RESULTS_TILE_POSITION),
+				<MediaCard
+					key="manyResultsTile"
+					id="manyResultsTileId"
+					type={null}
+					preview="/images/more.png"
+					title={tText(
+						'modules/shared/components/media-card-list/media-card-list___teveel-resultaten'
+					)}
+					view={view}
+				/>,
+				...tiles.slice(MANY_RESULTS_TILE_POSITION),
+			];
+		}
+		return tiles;
+	};
```

Comment:
We always want to show it at the end of the page.
The 10000 is the number of results that elasticsearch will want to output. But even if it is more, we always want to show that tile at the end of the page. So we don't need to do this extra work.
```suggestion
	const insertManyResultsTile = (tiles: ReactNode[]): ReactNode[] => {
		if (showManyResultsTile) {
			tiles.push(<MediaCard
					key="manyResultsTile"
					id="manyResultsTileId"
					type={null}
					preview="/images/more.png"
					title={tText(
						'modules/shared/components/media-card-list/media-card-list___teveel-resultaten'
					)}
					view={view}
				/>)
		}
		return tiles;
	};
```


FYI, if you ever need to insert an element into a specific position in an array, you can use the splice method instead of the slice method: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice